# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: unknown
# Commit: b83b3c1baa19
# Full Hash: b83b3c1baa19d3c914da2fa081b467da6fa6ba1f
# Author: Yoshi Cheng-Hao Huang <allstars.chh@gmail.com>
# Date: 2025-10-03 20:53 +0000
# Regressor Bug: 1968890
# File Overlap Count: 3
# Overlapping Files: dom/base/test/jsmodules/chrome.toml, dom/base/test/jsmodules/import_404.mjs, dom/base/test/jsmodules/test_dynamicImportErrorMessage.html
# Description:
#   Bug 1968890 - Part 9: Add test for dynamic import. r=jonco
# ==============================================================================

diff -r a80062ebc58c -r b83b3c1baa19 dom/base/test/jsmodules/import_404.mjs
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/dom/base/test/jsmodules/import_404.mjs	Fri Oct 03 20:53:30 2025 +0000
@@ -0,0 +1,2 @@
+/* eslint-disable import/no-unresolved */
+import { A } from "./404.js";